src/static/pretixcontrol/js/ui/main.js

```diff
@@ -143,7 +143,10 @@ $(function () {
                 clear: 'fa fa-trash',
                 close: 'fa fa-remove'
             }
-        });
+        };
+        if ($(this).data().hasOwnProperty("paymentWeekdaysDisabled"))
```

Comment:
That should tally with the above suggestion so yeah, I don't see any reason why not. Will update accordingly